spark/src/main/scala/io/delta/sql/parser/DeltaSqlParser.scala

```diff
@@ -368,7 +368,8 @@ class DeltaSqlAstBuilder extends DeltaSqlBaseBaseVisitor[AnyRef] {
     OptimizeTableCommand(
       Option(ctx.path).map(string),
       Option(ctx.table).map(visitTableIdentifier),
-      Option(ctx.partitionPredicate).map(extractRawText(_)).toSeq)(interleaveBy)
+      Option(ctx.partitionPredicate).map(extractRawText(_)).toSeq,
+      DeltaOptimizeContext(isFull = ctx.FULL != null))(interleaveBy)
```

Comment:
nit: Indent the tab space

Reply:
oh I meant DeltaOptimizeContext is part of the above line so we can indent it by a tab space

Reply:
@rahulsmahadev , I think the current indention is correct. `DeltaOptimizeContext(isFull = ctx.FULL != null) ` is not part of the above line and it is its own a new parameter as previous lines.

Reply:
ah ok I misunderstood